src/manifest/representation.ts

```diff
@@ -21,6 +21,7 @@ import {
   IContentProtections,
   IParsedRepresentation,
 } from "../parsers/manifest";
+import { IHDRInformation } from "../parsers/manifest/dash/get_hdr_information";
```

Comment:
I'm not sure about importing something from the DASH parser there:

To me:
  - Either `IHDRInformation` is defined in `src/manifest/...` and thus can be defined one time and imported by the various parsers
  - Either it is defined at both places
 
Thoughts?

Also, as we're speaking about properties which may be acceded through the API, maybe we want that structure's `colorDepth` property typed as `number|undefined` (like in the API documentation).

That way both TypeScript, and the possible exported type (if we decide to add it to there) would return the exact same format than the API.

Reply:
For the following reasons:
  1. I feel that there's something wrong when the resulting `Manifest` interface defined in the parsers, because formatting to a structure close to `Manifest` is the goal of those parsers. It should always be the other way around no?
  2. If that same interface is also parsed in another streaming protocol, which one should we depend on here?
  3. If that same interface is also parsed in another streaming protocol, we would have to mostly copy-paste that code.
  4. We're depending on a file that is feature-switchable. As this is only a type, it shouldn't bring the size penalty, but still

It's a little different for `IContentProtections` and `IParsedRepresentation` because they are both defined not in a specific parser file but for the interface type outputed by all of them, written in a specific "types.ts" common file.
Though it's true that the former (`IContentProtections`) may still suffer from `1`.